# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/resource-timing/supported_resource_type.any.js.ini
# Commit: 5248abfed275
# Full Hash: 5248abfed275b2938ed64770bec58b3ff68d624b
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-12 21:56:33
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=mstange,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27008
# ==============================================================================

diff -r e17500911824 -r 5248abfed275 testing/web-platform/meta/resource-timing/supported_resource_type.any.js.ini
--- a/testing/web-platform/meta/resource-timing/supported_resource_type.any.js.ini	Fri Apr 12 15:49:22 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,9 +0,0 @@
-[supported_resource_type.any.html]
-  [supportedEntryTypes contains 'resource'.]
-    expected: FAIL
-
-
-[supported_resource_type.any.worker.html]
-  [supportedEntryTypes contains 'resource'.]
-    expected: FAIL
-